src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Changed.cs

```diff
@@ -67,34 +64,33 @@ public void FileSystemWatcher_File_Changed_DataModification()
         [ConditionalFact(typeof(MountHelper), nameof(MountHelper.CanCreateSymbolicLinks))]
         public void FileSystemWatcher_File_Changed_SymLink()
         {
-            using (var testDirectory = new TempDirectory(GetTestFilePath()))
-            using (var dir = new TempDirectory(Path.Combine(testDirectory.Path, "dir")))
-            using (var file = new TempFile(GetTestFilePath()))
-            using (var watcher = new FileSystemWatcher(dir.Path, "*"))
+            string dir = CreateTestDirectory(TestDirectory, "dir");
+            string file = CreateTestFile();
+            using (var watcher = new FileSystemWatcher(dir, "*"))
             {
                 watcher.NotifyFilter = NotifyFilters.FileName | NotifyFilters.Size;
-                Assert.True(MountHelper.CreateSymbolicLink(Path.Combine(dir.Path, GetRandomLinkName()), file.Path, false));
+                Assert.True(MountHelper.CreateSymbolicLink(Path.Combine(dir, GetRandomLinkName()), file, false));
 
-                Action action = () => File.AppendAllText(file.Path, "longtext");
-                Action cleanup = () => File.AppendAllText(file.Path, "short");
+                Action action = () => File.AppendAllText(file, "longtext");
+                Action cleanup = () => File.AppendAllText(file, "short");
 
-                ExpectEvent(watcher, 0, action, cleanup, expectedPath: file.Path);
+                ExpectEvent(watcher, 0, action, cleanup, expectedPath: file);
             }
         }
 
         [Fact]
         public void FileSystemWatcher_File_Changed_SynchronizingObject()
         {
-            using (var testDirectory = new TempDirectory(GetTestFilePath()))
-            using (var file = new TempFile(Path.Combine(testDirectory.Path, "file")))
-            using (var watcher = new FileSystemWatcher(testDirectory.Path, Path.GetFileName(file.Path)))
+            string file = CreateTestFile(TestDirectory, "file");
+            File.Create(file).Dispose();
```

Comment:
```suggestion
```
you don't need this any more. I see this also in 3 other places.

Reply:
I couldn't find any more of these.